hub/load-test-http/README.md

```diff
@@ -1,83 +1,9 @@
-
-```shell
-iter8 run --set url=https://example.com \
-          --set SLOs.error-rate=0 \
-          --set SLOs.latency-mean=50 \
-          --set SLOs.latency-p90=100 \
-          --set SLOs.latency-p'97\.5'=200
-```
-
-***
-
-Set the number of requests sent during the load test to 200, the number of requests per second to 10, and the number of parallel connections used to send the requests to 5, as follows.
-
-```shell
-iter8 run --set url=https://example.com \
-          --set SLOs.error-rate=0 \
-          --set SLOs.latency-mean=50 \
-          --set SLOs.latency-p90=100 \
-          --set SLOs.latency-p'97\.5'=200 \
-          --set numQueries=200 \
-          --set qps=10 \
-          --set connections=5
-```
-
-***
-
-Supply a string as payload. This sets the content type to application/octet-stream.
-
-```shell
-iter8 run --set url=http://127.0.0.1/post \
-          --set payloadStr="abc123" \
-          --set SLOs.error-rate=0 \
-          --set SLOs.latency-mean=50 \
-          --set SLOs.latency-p90=100 \
-          --set SLOs.latency-p'97\.5'=200
-```
-
-***
-
-Fetch JSON content from a payload URL. Use this JSON as payload and explicitly set the content type to application/json.
-
-```shell
-iter8 run --set url=http://127.0.0.1/post \
-          --set payloadURL=https://data.police.uk/api/crimes-street-dates \
-          --set contentType="application/json" \
-          --set SLOs.error-rate=0 \
-          --set SLOs.latency-mean=50 \
-          --set SLOs.latency-p90=100 \
-          --set SLOs.latency-p'97\.5'=200
-```
-
-***
-
-
-By default, the following metrics are collected by `load-test-http`: `error-count`, `error-rate`, `latency-mean`, `latency-max`, `latency-stddev`, `latency-min`, and latency percentiles in the list [50.0, 75.0, 90.0, 95.0, 99.0, 99.9]. In addition, any other latency percentiles that are specified as part of SLOs are also collected.
-
-Consider the following command.
-```shell
-iter8 run --set url=https://example.com \
-          --set SLOs.error-rate=0 \
-          --set SLOs.latency-mean=50 \
-          --set SLOs.latency-p90=100 \
-          --set SLOs.latency-p'97\.5'=200
-```
-
-The above command ensures the following.
-
-1. The following latency percentiles are collected and reported: `[50.0, 75.0, 90.0, 95.0, 97.5, 99.0, 99.9]`.
-2. The following SLOs are validated.
-  * error rate is 0
-  * mean latency is under 50 msec
-  * 90th percentile latency is under 100 msec
-  * 97.5th percentile latency is under 200 msec
 
+- [Overview](https://iter8.tools/0.8/tutorials/load-test-http/overview/)
+- [Your first experiment](https://iter8.tools/0.8/getting-started/your-first-experiment/)
+- [Load characteristics](https://iter8.tools/0.8/tutorials/load-test-http/loadcharacteristics/)
+- [Request payload](https://iter8.tools/0.8/tutorials/load-test-http/payload/)
+- [Metrics and SLOs](https://iter8.tools/0.8/tutorials/load-test-http/metricsandslos/)
```

Comment:
The links have changed; should be updated.

Reply:
mkdocs has appropriate redirects for them. So, they are not broken links... will change the actual links after we cut this release.

Reply:
Please see README comment earlier on updating links prematurely before they come into existence.